airflow/contrib/hooks/sagemaker_hook.py

```diff
@@ -34,11 +36,15 @@ def __init__(self,
                  sagemaker_conn_id=None,
                  use_db_config=False,
                  region_name=None,
+                 check_interval=2,
```

Comment:
Change it to 5 which is used in python sdk.

Reply:
fixed